packages/ruleset/test/utils/root-document.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
         },
       },
     },
-    '/v1/drinks/menu': {
+    '/v1/drinkmenu': {
```

Comment:
It turns out we already had an example of ambiguous paths in root-document.js :)